src/backend/joanie/tests/lms_handler/test_backend_openedx.py

```diff
@@ -411,17 +411,20 @@ def test_backend_openedx_set_enrollment_with_related_certificate_product(self):
         )
 
         order.flow.cancel()
-
-        self.assertEqual(len(responses.calls), 4)
-        self.assertEqual(
-            json.loads(responses.calls[3].request.body),
-            {
-                "is_active": is_active,
-                "mode": "honor",
-                "user": user.username,
-                "course_details": {"course_id": "course-v1:edx+000001+Demo_Course"},
-            },
-        )
+        if enrollment.is_active is True:
```

Comment:
```suggestion
        if enrollment.is_active:
```
Could it be truthy ?

Reply:
Yup it could as this property is always a boolean value. Just in general I prefer to prevent the use a truthy statement which could lead to false positive.